index.html

```diff
@@ -0,0 +1,332 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio: Setup and Mobile First Web App</title>
+    <link rel="stylesheet" href="css/styles.css">
+    <!-----<link rel="stylesheet" href="css/finish.css"--------->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+
+    <div class="wrapper">
+        
+        <header class="header-2">
+            
+                <a href="./index.html" class="my-logo">My Logo</a>
+                <nav class="navbar">
+                    <ul>
+                        <li class="menu-bar">                            
+                            <button type="button" class="bar"><i class="fa fa-bars"></i></button>
+                        </li>
+                    </ul>
+                </nav>
+            
+        </header>
+        
+        <div class="container-homepage">     
+            <section id="#about" class="section-home">
```

Comment:
- [ ] Please reduce the space between your header section and the headline section and also add the background image to match the design template. This is because you are to stick to the design template as much as possible.
*Your design*
![1](https://user-images.githubusercontent.com/94551484/186515007-a6773c31-3e67-43cc-9ffc-3c76fdc4576a.png)

*Figma template*
![aaa](https://user-images.githubusercontent.com/94551484/186515034-b75c0fd3-b660-4861-a728-80d7062f0b8b.png)

Try removing or editing the grid on the `wrapper` element to fix the spacing problem.